python_src/src/lamp_py/postgres/seed_metadata.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import sys
+from typing import List
+
+import sqlalchemy as sa
+
+from lamp_py.runtime_utils.import_env import load_environment
+
+from .postgres_schema import (
+    MetadataLog,
+    SqlBase,
+    VehicleEventMetrics,
+    VehicleEvents,
+    VehicleTrips,
+)
+from .postgres_utils import DatabaseManager
+
+logging.getLogger().setLevel("INFO")
+
+DESCRIPTION = """Interact with Performance Manager RDS for Testing"""
+
+
+def parse_args(args: List[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+
+    parser.add_argument(
+        "--env-file",
+        dest="env_file",
+        help="environment file containing rds connection parameters",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        dest="verbose",
+        help="if set, use debug logging",
+    )
+
+    parser.add_argument(
+        "--seed-file",
+        dest="seed_file",
+        help="if set, read a json file to seed the metadata table with",
+    )
```

Comment:
Would be nice if this defaulted to to the existing json file, since we don't have more than one to seed with.

Reply:
i'm handling this in the docker compose file, since i'm adding the test files into the image with a volume. i'm hoping to eventually take the tests directory out of our image.